### PR TITLE
The TC Inflation Act of 2023

### PR DESCRIPTION
**Pull request by matttheficus**
## What Does This PR Do
Multiplies most TC costs, rewards, and totals by five.
`20TC` Traitor Uplinks > `100TC` etc etc etc.
Contractor payouts and totals multiplied by 5.
Nuclear Operatives total TC multiplied by 5 (and a smidge to make for an even number).
Uplink Items and such all multiplied by 5.

Additionally, see below.
### Nerfs/Buffs - % Cost Difference in Comparison to Old Cost
**Traitor Items:**
Bomber Jacket: `1TC` (-80%)
Camera Bug: `3TC` (-40%)
Cardboard Cutouts: `1TC` (-80%)
Cham Voice Mod Mask: `7TC` (+40%)
Chameleon Flag: `1TC` (-80%)
Chameleon Stamp: `1TC` (-80%)
Safecracking Kit: `1TC` (-80%)
DNA Scrambler: `5TC` (-50%)
False Bottom Briefcase: `10TC` (-33%)
Feral Cat Grenade: `10TC` (-33%)
FRAME Cartridge: `15TC` (-25%)
Singularity Beacon: `10TC` (-67%)

**Operative Items:**
Flamethrower: `3TC` (-40%)
MedKit: `30TC` (+50%)
Manhack Delivery: `10TC` (-60%)
Eshield: `40TC` (-50%)
Grenade Belt: `120TC` (-20%)
Assault Borg: `225TC` (-32%)
Gygax: `400TC` (-11%)
Mauler: `600TC` (-14%)

## Why It's Good For The Game
This has been discussed by most of the Dev Team for a few months now. This will allow for more variance in Uplink/Operative items in the future, as well as allow us to fine-tune items that are over/underperforming in a more slight way.

This will lead to people having to add bigger numbers together. I do hope everyone is capable of basic math. If not, your phone or computer has a calculator if you cant count to 100 (Operatives on the other hand get a LOT more TC to work with, so I wont fault you there).

In the future, we've discussed adding a "checkout" system, especially for Operatives, so that you can add stuff to a "cart" and check out your items in bulk, perhaps with some discounts that way for Traitors too. But, as that touches TGUI and would build upon this, Ill leave that for later/someone else to do it. 

## Testing
1. Spawn in.
2. Purchase warcrimes from various uplinks.
3. Double check that Contracts are paid out correctly.

## Changelog
:cl:
tweak: Tweaked TC economy to work from a 100TC budget for Traitors and radiating changes to all other Uplink systems.
/:cl:

### CODE REVIEW

**Comment by Coolrune206** [comment]
I suggest leaving the E-Swords somewhere above 30 TC, even if you lower the price, so you can't buy DESword adrenals without any discounts. Especially considering the buffs DESwords (and adrenals) have gotten recently.

**Comment by matttheficus** [comment]
Esword moved to `35TC` due to buffs recently. You can still kinda destroy this item with a baton so the Dev team discussion on it being a bit weaker overall is still applied IMO.

**Comment by GDNgit** [comment]
> Esword moved to `35TC` due to buffs recently. You can still kinda destroy this item with a baton so the Dev team discussion on it being a bit weaker overall is still applied IMO.

if you attempt to slap a dsword user with a baton now and they parry your ass gets send to the slowdown zone
not a steller idea

File: code/game/gamemodes/nuclear/nuclear_challenge.dm [hunk 1]
```diff
@@ -1,7 +1,7 @@
-#define CHALLENGE_TELECRYSTALS 280
+#define CHALLENGE_TELECRYSTALS 1150
 #define CHALLENGE_TIME_LIMIT 6000
 #define CHALLENGE_SCALE_PLAYER 1 // How many player per scaling bonus
-#define CHALLENGE_SCALE_BONUS 2 // How many TC per scaling bonus
+#define CHALLENGE_SCALE_BONUS 8 // How many TC per scaling bonus
 #define CHALLENGE_MIN_PLAYERS 50
 #define CHALLENGE_SHUTTLE_DELAY 18000 //30 minutes, so the ops have at least 10 minutes before the shuttle is callable. Gives the nuke ops at least 15 minutes before shuttle arrive.
```
Comment (hal9000PR) [inline code/game/gamemodes/nuclear/nuclear_challenge.dm:6]: 280 * 5 is 1400
2 * 5 is 10
these nerfs werent mentioned in the PR

**Review by hal9000PR — CHANGES_REQUESTED** [review]

File: code/game/gamemodes/nuclear/nuclear_challenge.dm (hunk 1 above)
Comment (matttheficus) [reply code/game/gamemodes/nuclear/nuclear_challenge.dm:6]: Maths issue, whoops.

**Comment by matttheficus** [comment]
Gonna close and reopen a PR that has none of the Balance changes in it.

TC change should come before the Balance shit. Will get this open in the next day or two.

**Comment by Henri215** [comment]
> Gonna close and reopen a PR that has none of the Balance changes in it.
> 
> TC change should come before the Balance shit. Will get this open in the next day or two.

image

**Pull request closed without merging**